src/main/java/edu/iis/mto/time/Order.java

```diff
@@ -24,11 +25,15 @@ public void addItem(OrderItem item) {
 
 	}
 
-	public void submit() {
+	public void submit(DateTime dateTimeTest) {
```

Comment:
To nie jest dobre rozwiązanie. Parametr metody nie może być logicznie oparty o "sposób uruchomienia" i testowany na istnienie z tego powodu.